src/lib/components/Grid.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import Lightbox from "./Lightbox";
 import "./Grid.css";
 
-const Grid = ({ images, rowHeight, margin = 0, width }) => {
+const Grid = ({ images, rowHeight, margin = 0, width, lightbox = true }) => {
```

Comment:
Would prefer `isLightboxEnabled` for this :)

Reply:
Sure :)